packages/js/src/ui/components/Renderer.tsx

```diff
@@ -20,7 +20,7 @@ export const novuComponents = {
   Bell,
 };
 
-export type NovuComponent = { name: NovuComponentName; props?: unknown };
+export type NovuComponent = { name: NovuComponentName; props?: any };
```

Comment:
Please add stronger types as the union of the props of all available components.

Reply:
Unfortunately, this can't be determined by the places where this type is used. Other types don't work well.